coordinape/coordinape
# Adds Fixed Payment Token type

Fixes #808

### Test Plan

1. go to http://localhost:3000/admin/circles
2. Click on Settings button
3. Go to Fixed Payment Token field and select a new value
4. click on save and ensure the value is saved in the backend
5. Make sure you cannot save any value other than the values presented in the dropdown.
